### PR TITLE
InstData Import: use names instead of library object for steps interacting

### PR DESCRIPTION
**Pull request by ebelter**
with gmt picard fastq to sam

### CODE REVIEW

File: lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/SraToBam.pm [hunk 1]
```diff
@@ -288,7 +292,9 @@ sub convert_fastq_to_bam {
         my $fastq_to_sam = Genome::Model::Tools::Picard::FastqToSam->create(
             fastq           => "$unaligned_fastq",
             output          => "$unaligned_bam",
-            sample_name     => "$sample_name",
+            sample_name => $self->sample_name,
+            library_name => $self->library_name,
+            read_group_name => $self->library_name,
```
Comment (acoffman) [inline lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/SraToBam.pm]: I see that this class doesn't have a `$self->read_group_name`, so I'm assuming the re-use of `library_name` is intentional here?

Comment (ebelter) [inline lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/SraToBam.pm]: Yes - the library_name here goes into the LB tag in the RG header. SM is sample_name. Ex:

@RG	ID:__TEST_SAMPLE__-extlibs	LB:__TEST_SAMPLE__-extlibs	SM:__TEST_SAMPLE__

Comment (acoffman) [inline lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/SraToBam.pm]: Cool, makes sense to me!

**Comment by acoffman** [comment]
+1

**Comment by tmooney** [comment]
+1

**Pull request merged**